config/rbac/manager_role.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
```

Comment:
```suggestion
```

Reply:
Auto-generated file so I can't remove this unfortunately